### PR TITLE
Classical data file (LDA)

### PR DESCRIPTION
**Pull request by wonagram**
I upload file for classical data collecting that I mentioned in slack. 
This file only use LDA to get energy densities. 
I will change it to cover pw92 later

### CODE REVIEW

**Comment by minhtriet** [comment]
If you'd like, you can separate the training and inference loop as in #28. There are some components there you can reuse

File: graddft_qnn/Classical.py [hunk 1]
```diff
@@ -0,0 +1,339 @@
...
+    # Molecule can compute the density matrix.
+    rho = jnp.clip(molecule.density(), a_min=clip_cte)
+    # Now we can implement the LDA energy density equation in the paper.
+    lda_e = -3/2 * (3/(4*jnp.pi)) ** (1/3) * (rho**(4/3)).sum(axis = 1, keepdims = True)
+    pw92_corr_e = pw92_densities(molecule, clip_cte)
+    # For simplicity we do not include the exchange polarization correction
+    # check function exchange_polarization_correction in functional.py
+    # The output of features must be an Array of dimension n_grid x n_features.
+    #print(f"LDA Energy Density - Shape: {lda_e.shape}, Size: {lda_e.size}")
+    return lda_e
```
Comment (jackbaker1001) [inline graddft_qnn/Classical.py:60]: Why is this returning just lda_e?

**Review by jackbaker1001 — APPROVED** [review]

File: graddft_qnn/Classical_No_Down.py [hunk 2]
```diff
@@ -0,0 +1,332 @@
...
+    # Molecule can compute the density matrix.
+    rho = jnp.clip(molecule.density(), a_min=clip_cte)
+    # Now we can implement the LDA energy density equation in the paper.
+    lda_e = -3/2 * (3/(4*jnp.pi)) ** (1/3) * (rho**(4/3)).sum(axis = 1, keepdims = True)
+    pw92_corr_e = pw92_densities(molecule, clip_cte)
+    # For simplicity we do not include the exchange polarization correction
+    # check function exchange_polarization_correction in functional.py
+    # The output of features must be an Array of dimension n_grid x n_features.
+    #print(f"LDA Energy Density - Shape: {lda_e.shape}, Size: {lda_e.size}")
+    return jnp.concatenate((lda_e,pw92_corr_e), axis=1)
```
[the remark this answers is not in the document]
Comment (wonagram) [reply graddft_qnn/Classical_No_Down.py:60]: I supposed to make it `return lda_e`, and `return jnp.concatenate((lda_e,pw92_corr_e), axis=1)` also working. 
But later code doesn't involves weighted sum with coefficients. Should be updated after HK's PR is done
